File: loop_transformations/sources/apply_nested.1.c
/*
* @@name:	apply_nested.1
* @@type:	C
* @@operation:	compile
* @@expect:	success
* @@version:	omp_6.0
*/
void nested_apply(double A[100])
{
    #pragma omp tile sizes(10) \
            apply(intratile: unroll partial(2) apply(reverse))
    for (int i = 0; i < 100; ++i)
        A[i] = A[i] + 1;
}

File: loop_transformations/sources/apply_nested.1.f90
! @@name:       apply_nested.1
! @@type:       F-free
! @@operation:  compile
! @@expect:     success
! @@version:    omp_6.0
subroutine nested_apply(A)
   implicit none
   double precision :: A(0:99)
   integer          :: i

   !$omp tile sizes(10) apply(intratile: unroll partial(2) apply(reverse))
   do i = 0, 99
      A(i) = A(i) + 1 
   enddo
end subroutine
